app/src/main/java/com/github/factotum_sdp/factotum/ui/directory/ContactsRecyclerAdapter.kt

```diff
@@ -52,9 +50,9 @@ class ContactsRecyclerAdapter (private val originalContacts: List<ContactsList.C
         return ContactsViewHolder(v)
     }
 
+    @SuppressLint("SetTextI18n")
```

Comment:
Is this @SuppressLint("SetTextI18n") really useful, from my understanding it's for "Internationalization" and is used way you have regional differences.